### ISSUE TITLE
Update 2.17 to 2.2.1 did not work for me

### ISSUE THREAD

**Issue by stone1978** [threema-ch/threema-web#907]
Hello

I updated my Threema-Web Site from 2.17 to 2.2.1

unzipped the files and copied them on the webspace. overwrote existing files.

as i always did until 2.1.7. that always worked.

now I get an empty page without QRCODE

https://cloud.stonesblog.at/threema-web/
LOOK here

Any Ideas ?

Regards Gerd

**Comment by lgrahl** [comment]
The web server needs to set the proper MIME type for `*.wasm` (WebAssembly) files, see: https://github.com/threema-ch/threema-web/blob/master/docs/self_hosting.md

**Comment by stone1978** [comment]
Is this new ?
OK i will read that and try.
I have apache2 runing

regards

**Comment by stone1978** [comment]
YES ;-)
https://stone-cloud/threema-web/#!/welcome
It Works Thank you

Solution for APACHE2
EDIT (VIM or NANO) /etc/apache2/mods-enabled/mime.conf

in de Application Section
ADD this LINE

AddType application/wasm    .wasm

SAVE

sudo service apache2 restart

try

it worked 

Thank you :)

**Comment by dbrgn** [comment]
> Is this new ?

Yes, and it was mentioned in the [release notes for 2.2.0](https://github.com/threema-ch/threema-web/blob/master/CHANGELOG.md) :slightly_smiling_face: It's always a good idea to take a look at those when upgrading.

**Comment by stone1978** [comment]
👍
